tests/unit/api.py

```diff
@@ -1196,6 +1197,19 @@ def test_list_projects(self, db):
             'runway',
         ]
 
+    def test_list_networks(self, db):
+        assert json.loads(api.list_networks()) == []
+        api.project_create('anvil-nextgen')
+        network_create_simple('eth0', 'anvil-nextgen')
+        assert json.loads(api.list_networks()) == ['eth0']
+        network_create_simple('wlan0', 'anvil-nextgen')
+        network_create_simple('wlan1', 'anvil-nextgen')
+        assert sorted(json.loads(api.list_networks())) == [
+            'eth0',
+            'wlan0',
+            'wlan1',
```

Comment:
These names sound like NICs. Could you change them to be something more networky, like "netA"?